iroh-net/src/relay/http/client.rs

```diff
@@ -254,6 +258,13 @@ impl ClientBuilder {
         self
     }
 
+    /// Sets whether to connect to the relay via websockets or not.
+    /// Set to use non-websocket, normal relaying by default.
```

Comment:
I think it would be great if we could give some guidance here to the user what this means.  Why would a user choose websockets?  What happens when the server doesn't support it?

As written this is tells you something if you know this feature, but you need to explain the feature to the user because this is the first time they're encountering this and they won't have a clue what to do with this.  It's perfectly fine if that means repeating lots of stuff available elsewhere.

Reply:
You're right. At the moment I'd say this API is mostly for testing, or *maybe* if you have an http proxy that blocks http upgrades except for websockets. (Is something like that a thing?)

I'll update the docs :v: